date/test.js

```diff
@@ -103,32 +103,73 @@ describe("get day of week by date", function() {
         assert.equal(dayOfWeek('07.05.2016'), 'Saturday');
     });
 
-    it(`13.05.2016 is Friday`, function() {
+    it('13.05.2016 is Friday', function() {
         assert.equal(dayOfWeek('13.05.2016'), 'Friday');
     });
-    it(`16.06.1910 is Thursday`, function() {
-        assert.equal(dayOfWeek('13.05.2016'), 'Thursday');
+    it('16.06.1910 is Thursday', function() {
+        assert.equal(dayOfWeek('16.06.1910'), 'Thursday');
     });
-    it(`11.10.2006 is Wednesday`, function() {
-        assert.equal(dayOfWeek('13.05.2016'), 'Wednesday');
+    it('11.10.2006 is Wednesday', function() {
+        assert.equal(dayOfWeek('11.10.2006'), 'Wednesday');
     });
-    it(`17.09.2019 is Tuesday`, function() {
-        assert.equal(dayOfWeek('13.05.2016'), 'Tuesday');
+    it('17.09.2019 is Tuesday', function() {
+        assert.equal(dayOfWeek('17.09.2019'), 'Tuesday');
     });
-    it(`04.01.1999 is Monday`, function() {
-        assert.equal(dayOfWeek('13.05.2016'), 'Monday');
+    it('04.01.1999 is Monday', function() {
+        assert.equal(dayOfWeek('04.01.1999'), 'Monday');
     });
     describe("throw exception on invalid value", function() {
 
-        let invalidValue = ['12,2.01.1234','12.32.1006','32.2.00','','07.07.-1006','07.07.-1006','12..2.08','0.-.=','   '];
-        function makeTest(x) {
+        let dateInvalidValue = [''];
+        function makeTestForDate(x) {
+            it(`${x} throws exception`, function() {
+                assert.throws(() => dayOfWeek(x), 'Invalid value of date');
+            });
+        }
+
+        for (let i = 0;
+            i < dateInvalidValue.length; i++) {
+            makeTestForDate(dateInvalidValue[i]);
+        }
+
+
+        let dayInvalidValue = ['12,2.01.1234', '32.2.00', '     ', '0.-.='];
```

Comment:
I read the mocha documentation and found a more easier and shorter way to write parameterized tests (actualy I can figure out by my own):
```javascript
['12,2.01.1234', '32.2.00', '     ', '0.-.='].forEach(x => {
       it(`${x} throws exception`, function() {
                assert.throws(() => dayOfWeek(x), 'Invalid value of day');
        });
})
```

Reply:
Im not sure I did this properly, tests broke down after my fix(